pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -393,6 +429,8 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 		{
 			ObjectMeta: kapi.ObjectMeta{
 				Name: BasicUserRoleName,
+				Annotations: map[string]string{
+					oapi.OpenShiftDescription: "A user that can get basic information about projects."},
```

Comment:
what does this even mean? they can Get/List the project but not get any objects within the project?